tests/src/test/scala/com/nvidia/spark/rapids/BroadcastHashJoinSuite.scala

```diff
@@ -35,13 +37,14 @@ class BroadcastHashJoinSuite extends SparkQueryCompareTestSuite {
       val df5 = df4.join(df3, Seq("longs"), "inner")
 
       val plan = df5.queryExecution.executedPlan
```

Comment:
sorry, just one more nit. It would be better to get the reference to the executedPlan after calling collect rather than before. The goal is to get the final plan, so it would be clearer in this order. It isn't obvious to people looking this code that the executedPlan gets updated by the call to collect. Other than this, LGTM.